src/Storage/Device/Local.php

```diff
@@ -69,24 +71,70 @@ public function getPath($filename): string
      *
      * @param string $source
      * @param string $path
+     * @param int $chunk
+     * @param int $chunks
+     * @param string tmp
      *
      * @throws \Exception
      *
-     * @return bool
+     * @return int
      */
-    public function upload($source, $path): bool
+    public function upload($source, $path, $chunk = 1, $chunks = 1): int
     {
         if (!\file_exists(\dirname($path))) { // Checks if directory path to file exists
             if (!@\mkdir(\dirname($path), 0755, true)) {
                 throw new Exception('Can\'t create directory: ' . \dirname($path));
             }
         }
 
-        if (\move_uploaded_file($source, $path)) {
-            return true;
-        }
+        //move_uploaded_file() verifies the file is not tampered with
+        if($chunks == 1) {
+            if (!\move_uploaded_file($source, $path)) {
+                throw new Exception('Can\'t upload file ' . $path);
+            }
+            return $chunks;
+        } else {
```

Comment:
Do we need the else block? we already use return, we can have 1 indentation less here.

Reply:
not sure why you see the old code, else was already removed.